Jenkinsfile

```diff
@@ -38,12 +39,10 @@ pipeline {
         echo "Building ${env.SERVICE} docker image"
 
         // Docker build flags are set via the getDockerBuildFlags() shared library.
-        sh "docker build ${getDockerBuildFlags()} -t rightbrainnetworks/auto-semver:${env.VERSION} ."
-
-
-        sh "python setup.py sdist"
- 
-        stash includes: "dist/semver-${env.SEMVER_NEW_VERSION}.tar.gz", name: 'PACKAGE' 
+        script
```

Comment:
you dont need script here

Reply:
Required for `dockerImage` and `docker.build`